catalyst/utils/torch.py

```diff
@@ -15,6 +15,13 @@
 from catalyst.tools.typing import Device, Model, Optimizer
 from catalyst.utils.dict import merge_dicts
 
+try:
+    import torch_xla.core.xla_model as xm
+
+    XLA_DEVICE = xm.xla_device()
```

Comment:
could we move `IS_XLA_AVAILABLE` to `catalyst/tools/settings`?
@bagxi what do you tnink?

Reply:
We can do so. But keep in mind that settings responsible only for imports (e.g. kornia) for now